core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -2375,7 +2375,11 @@ class WorkspaceService(protected val userInfo: UserInfo,
 
       billingProject = maybeBillingProject.getOrElse(throw new RawlsExceptionWithErrorReport(errorReport = ErrorReport(StatusCodes.BadRequest, s"Billing Project ${billingProjectName} does not exist")))
       _ <- failUnlessBillingProjectReady(billingProject)
-      _ <- failUnlessBillingAccountHasAccess(billingProject, parentSpan)
+      _ <- billingProject.billingProfileId match {
+        // If `billingProfileId` is defined, this project is backed by a BPM record, and we don't need to check access.
+        case Some(_) => Future.successful()
+        case _ => failUnlessBillingAccountHasAccess(billingProject, parentSpan)
```

Comment:
Exposed this method and modified it to work with both GCP and Azure billing projects.